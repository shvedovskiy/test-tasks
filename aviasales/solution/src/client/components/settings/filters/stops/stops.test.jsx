/* eslint-disable no-undef */
import React from 'react';

import Stops from './stops';


describe('<Stops />', () => {
  const props = {
    stops: {
      '0': true,
      '1': false,
      '2': false,
    },
    changeStops: jest.fn(),
  };
  const stops = shallow(<Stops {...props} />);

  it('renders properly', () => {
    expect(stops).toMatchSnapshot();
  });
});
