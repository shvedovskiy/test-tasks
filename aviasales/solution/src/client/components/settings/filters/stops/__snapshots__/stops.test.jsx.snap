// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stops /> renders properly 1`] = `
<styled__StopsList>
  <styled__StopsListItem>
    <styled__Label
      htmlFor="stops-all"
    >
      <StopsCheckbox
        checked={false}
        id="stops-all"
        onChange={[Function]}
        value="-1"
      />
      <styled__LabelText>
        Все
      </styled__LabelText>
    </styled__Label>
  </styled__StopsListItem>
  <styled__StopsListItem
    key="0"
  >
    <styled__Label
      htmlFor="stops-0"
    >
      <StopsCheckbox
        checked={true}
        id="stops-0"
        onChange={[Function]}
        value="0"
      />
      <styled__LabelText>
        Без пересадок
      </styled__LabelText>
    </styled__Label>
    <StopOnly
      handleSelect={[Function]}
    />
  </styled__StopsListItem>
  <styled__StopsListItem
    key="1"
  >
    <styled__Label
      htmlFor="stops-1"
    >
      <StopsCheckbox
        checked={false}
        id="stops-1"
        onChange={[Function]}
        value="1"
      />
      <styled__LabelText>
        1 пересадка
      </styled__LabelText>
    </styled__Label>
    <StopOnly
      handleSelect={[Function]}
    />
  </styled__StopsListItem>
  <styled__StopsListItem
    key="2"
  >
    <styled__Label
      htmlFor="stops-2"
    >
      <StopsCheckbox
        checked={false}
        id="stops-2"
        onChange={[Function]}
        value="2"
      />
      <styled__LabelText>
        2 пересадки
      </styled__LabelText>
    </styled__Label>
    <StopOnly
      handleSelect={[Function]}
    />
  </styled__StopsListItem>
</styled__StopsList>
`;
